src/libraries/System.CodeDom/src/System/CodeDom/Compiler/ExposedTabStringIndentedTextWriter.cs

```diff
@@ -47,14 +49,16 @@ internal string IndentationString
 
                     switch (_indent)
                     {
-                        case 0: _s = string.Empty; break;
```

Comment:
Why was this removed?

Reply:
It was dead code as far as I could see. `Indentation` is only called from `CSharpCodeGenerator` and `VBCodeGenerator`:
```cs

```cs
private string QuoteSnippetStringCStyle(string value)
{
    var b = new StringBuilder(value.Length + 5);

    var indentObj = new Indentation(_output, Indent + 1);
    ...
}
```

`Indent` is a wrapper around `IndentedTextWriter.Indent`
```cs
private int Indent
{
    get => _output.Indent;
    set => _output.Indent = value;
}
```

and `IndentedTextWriter.Indent` can never be less than zero:
```cs
public int Indent
{
    get { return _indentLevel; }
    set { _indentLevel = Math.Max(value, 0); }
}
```

Therefore, `Indent + 1` can never be zero